### PR TITLE
pkp/pkp-lib#2832 remove citationFormats in code

### PR DESCRIPTION
**Pull request by bozana**
s. here: https://github.com/pkp/pkp-lib/issues/2832

### CODE REVIEW

File: dbscripts/xml/upgrade/3.1.0_update.xml [hunk 1]
```diff
@@ -89,4 +89,8 @@
 		<query>UPDATE metrics SET assoc_type = 531 WHERE assoc_type = 515 AND assoc_id IN (SELECT file_id FROM submission_supplementary_files)</query>
 		<query>UPDATE metrics SET assoc_type = 531 WHERE assoc_type = 515 AND assoc_id IN (SELECT file_id FROM submission_artwork_files)</query>
 	</sql>
+	<sql><!-- pkp/pkp-lib#2832 -->
+		<!-- remove citationFormats plugins -->
+		<query>DELETE FROM versions WHERE product_type = 'plugins.citationFormats'</query>
```
Comment (NateWr) [inline dbscripts/xml/upgrade/3.1.0_update.xml]: there were also plugins under a category "citationOutput" which I believe is no longer being used.

Comment (bozana) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: I think those belong to the Citation Assistant -- I am not sure if Citation Assistant is still used :-\ However the citationOutput plugin is now in pkp-lib, so I would leave it in the DB as well for now. OK?

Comment (NateWr) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: Oh I see. I removed `citationOutput`. I think that's just the Czech local files again in `pkp-lib`: https://github.com/pkp/pkp-lib/tree/master/plugins/citationOutput

Comment (bozana) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: Oh, I see. Hmmm... Shall it then also be removed from the plugins folder and from the DB?

Comment (NateWr) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: Yeah I think so. I removed it all with the switch to CSL.

Comment (bozana) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: Hmmm... For some reason I am not sure about deleting citationOutput plugin :-\ Those filters are also still in the DB and also used/assumed that exist in other code :-\ I am not sure if that code is still relevant, but it is still there :-\ Then actually all this should be removed, no?

Comment (NateWr) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: I think @asmecher needs to weigh in on this. I'm not familiar with any of the previous tools that existed. I just cleaned out the stuff I saw that didn't appear to be used anymore. 

It looks like there's more stuff to be remove, but I don't know what impact that will have.

Comment (bozana) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: @NateWr, I would then maybe consider those citationOutput removal things in a new issue, if you do not mind -- I would like to investigate first what everything I could and should remove... OK?

Comment (NateWr) [reply dbscripts/xml/upgrade/3.1.0_update.xml]: Sounds good! :+1: That was the only comment I had so you can merge this when you'd liike.

**Pull request merged**